python_package/madflow/custom_op_generator.py

```diff
@@ -0,0 +1,165 @@
+import subprocess
+
+
+import madflow.wavefunctions_flow
+from madflow.makefile_template import write_makefile
+
+from madflow.op_write_templates import *
+from madflow.op_syntax import *
+from madflow.op_read import *
+from madflow.op_generation import *
```

Comment:
You should never import everything with a module with * it's way better to do just

`import madflow.op_write_templates as op_wt` (or whatever) and then use `op_wt.function_from_module`

otherwise if you by mistake put in `op_generation` a variable with the same name as something in `op_write_templates` everything will fail in a very non obvious way.

Reply:
Done.